ethcore/evm/Cargo.toml

```diff
@@ -21,3 +21,5 @@ rustc-hex = "1.0"
 
 [features]
 jit = ["evmjit"]
+evm-debug = []
+evm-debug-tests = ["evm-debug"]
```

Comment:
why introducing a new feature? is it used anywhere else? I can't see

Reply:
It is part of `evm` crate now and used to be part of `ethcore` - the feature was not correctly migrated when those two were split. It prints out VMtracing when running interpreter tests and is quite helpful.